src/contracts/pods/EigenPod.sol

```diff
@@ -616,6 +619,7 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         require(amountToWithdraw <= nonBeaconChainETHBalanceWei,
             "EigenPod.withdrawnonBeaconChainETHBalanceWei: amountToWithdraw is greater than nonBeaconChainETHBalanceWei");
         nonBeaconChainETHBalanceWei -= amountToWithdraw;
+        emit NonBeaconChainETHWithdrawn(recipient, amountToWithdraw);
         AddressUpgradeable.sendValue(payable(recipient), amountToWithdraw);
```

Comment:
we are not sending this through DWR?

Reply:
hmm, good callout. It might be "safer "to route it via the DWR, but I do feel like this is a fairly well-contained pair of functions. any particular concerns? If you are strongly opinionated and want to make a PR changing this behavior to route through the DWR instead, I'm fine with it.

Reply:
I prefer DWR for monitoring and cohesiveness of invariants. Wdyt @Sidu28

Reply:
Sounds good, will implement in a separate PR associate with this ticket: https://github.com/Layr-Labs/eignlayr-contracts/issues/962